modules/serdes/tests/tests.cpp

```diff
@@ -37,6 +41,25 @@ struct Data {
   int a = NUMBER;
 };
 
+struct DummyJSONSerializable {
+  auto operator==(const DummyJSONSerializable& other) const -> bool = default;
```

Comment:
```suggestion
  auto operator==(const DummyJSONSerializable&) const -> bool = default;
```
nit. Can be omitted.